opencensus/proto/stats/metrics/metrics.proto

```diff
@@ -69,15 +70,18 @@ message MetricDescriptor {
   Type type = 4;
 
   // TODO: Consider to add LabelInfo{key, description} if needed.
+  // The label keys associated with the metric descriptor.
+  repeated string label_keys = 5;
 }
```

Comment:
Do we want to specify an order? Probably not given our 'passthrough' philosophy but it is an issue we've discussed before.

Reply:
I thought the decision is not to do any validation here?

From @g-easy : 
> We want to avoid performing validation as much as we can when dealing with Metrics.

Reply:
No ordering please.

Reply:
sgtm